example/profiling.html

```diff
@@ -4,6 +4,14 @@
   <meta charset="utf-8">
   <title>Zones Profiling</title>
   <link rel="stylesheet" href="css/style.css">
+  <script>
+    __Zone_disable_Error = true;
+    __Zone_disable_on_property = true;
+    __Zone_disable_geolocation = true;
+    __Zone_disable_toString = true;
+    __Zone_disable_blocking = true;
+    __Zone_disable_PromiseRejectionEvent = true;
+  </script>
```

Comment:
@mhevery , only the names, should we named those flags as __zone_symbol__disable_toString?